### PR TITLE
Collect reference number when recording appointment summary

### PR DESCRIPTION
**Pull request by stevenwilkin**
image

### CODE REVIEW

File: app/models/appointment_summary.rb [hunk 1]
```diff
@@ -10,4 +10,5 @@ class AppointmentSummary < ActiveRecord::Base
   validates :income_in_retirement, inclusion: { in: %w(pension other) }
   validates :guider_name, presence: true
   validates :guider_organisation, inclusion: { in: %w(tpas cab) }
+  validates :reference_number, presence: true
```
Comment (andrewgarner) [inline app/models/appointment_summary.rb]: The database field is an integer so it would make sense to validate the numericality too.

Comment (stevenwilkin) [inline app/models/appointment_summary.rb]: Good call

**Comment by stevenwilkin** [comment]
Updated with validation to ensure the entered string parses as a number. This is intended to be merged in `master` now so 9ac53fa may need to be cherry-picked.

**Comment by stevenwilkin** [comment]
@aduggin will drop in the changes to the output document to include the reference number

**Pull request closed without merging**